kong/plugins/datadog/schema.lua

```diff
@@ -73,8 +73,8 @@ return {
     { config = {
         type = "record",
         fields = {
-          { host = typedefs.host({ default = "localhost" }), },
-          { port = typedefs.port({ default = 8125 }), },
+          { host = typedefs.host({ referenceable = true, default = "localhost" }), },
+          { port = typedefs.port({ referenceable = true, default = 8125 }), },
```

Comment:
let's not do the number change for now, the default value being null issue happens only on string field.